project.clj

```diff
@@ -4,9 +4,12 @@
   :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
             :url  "https://www.eclipse.org/legal/epl-2.0/"}
   :dependencies [[org.clojure/clojure "1.10.0" :scope "provided"]
-                 [org.slf4j/slf4j-log4j12 "1.7.32"]
+                 [com.fzakaria/slf4j-timbre "0.3.21"]
                  [com.taoensso/timbre "5.1.0"]
                  [de.active-group/active-clojure "0.40.0"]
                  [de.active-group/timbre-riemann "0.2.0"]
                  [cheshire "5.10.1"] ;; for logstash 3rd-party-appender of timbre
-                 [riemann-clojure-client "0.5.1" :exclusions [org.slf4j/slf4j-api]]])
+                 [riemann-clojure-client "0.5.1" :exclusions [org.slf4j/slf4j-api]]
```

Comment:
I think we could zap this exclusion it does not seem to make sense here. `slf4j-api` should not hurt.

Reply:
Fixed in 54e1bdf48de0e0f4f0ef3d95d2ede1465523c4c3.  I've decided to bump `riemann-clojure-client` while we're at it.